.github/workflows/publish-developer-edition.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   bump-version:
-    if: (github.event_name == 'schedule' || github.event_name == 'repository_dispatch' || github.actor == 'ronkelementor' || github.actor == 'KingYes' || github.actor == 'shilo-ey' || github.actor == 'matipojo') && startsWith(github.repository, 'elementor/')
+    if: (github.event_name == 'schedule' || github.event_name == 'repository_dispatch' || github.actor == 'yotamselementor' || github.actor == 'KingYes' || github.actor == 'TzviRabinovitch' || github.actor == 'matipojo') && startsWith(github.repository, 'elementor/')
```

Comment:
do we really have to make it hard coded?
can't we use something like this?
https://github.com/elementor/elementor/blob/master/.github/workflows/elementor-tests.yml#L13-L16

Reply:
We can, but I want to re-examine this entire process, it seems strange to me

Reply:
For now, I just wanted to be able to create a release